packages/pigeon/platform_tests/alternate_language_test_plugin/android/src/main/java/com/example/alternate_language_test_plugin/AlternateLanguageTestPlugin.java

```diff
@@ -273,6 +278,28 @@ public void reply(Void value) {
         });
   }
 
+  // TODO: (tarrinneal) Update error handling tests to properly recieve and handle errors.
+  // See issue https://github.com/flutter/flutter/issues/118243
```

Comment:
Hm, I guess we'll need a breaking change for Java. I was thinking we would convert channel errors to `throw`s, but that doesn't work since they are async, so we'll need to replace `Reply` with `Result` in these signatures.